frontend-reference/setting-goodie-display.md

```diff
@@ -229,6 +141,26 @@ return $plaintext,
 
 ```
 
+For an example of how this works, take a look at the final return statement of the [BPM to ms](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/BPMToMs.pm) Goodie Perl file:
+
+```perl
+return $plaintext,
+	structured_answer => {
+	    data => \@items, # In this case an array, but could be a hash (in the case of a single item)
+	    meta => {
+	        sourceUrl => "https://wikipedia.org/wiki/Tempo#Beats_per_minute",
+	        sourceName => "Wikipedia"
+	    },
+	    templates => {
+	        group => 'base',
```

Comment:
I don't think we want to use an IA that is using the base template here - people might get the wrong idea. Something using text or info would probably be best.